pkgs/top-level/all-packages.nix

```
@@ -2381,6 +2381,8 @@ with pkgs;
 
   borgmatic = callPackage ../tools/backup/borgmatic { };
 
+  borg-sans-mono_ttf = callPackage ../data/fonts/borg-sans-mono { };
```

```suggestion
  borg-sans-mono = callPackage ../data/fonts/borg-sans-mono { };
```

I'd just call it `borg-sans-mono`, there's only a handful of files that use `_ttf`, some of which have other versions

---

I was actually confused about that, I tried just to follow some example first, but you're right.

---

I don't know the ins-and-outs of font convention but yeah since it's only a handful.

I followed an example that puts fonts under `$out/share/fonts/ttf` rather than `$out/share/fonts/truetype` but it looks like `$out/share/fonts/truetype` is the most common

---

the rest looks good now, can you just fix the commit and PR message to `borg-sans-mono`

---

Done! Thx again for the review and for the quickness.

---

thanks. Can you also change the pname to `borg-sans-mono`, sorry I missed it

---

haha don't worry, done! Also changed the commit message accordingly.